src/APL/vercelAPL.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable class-methods-use-this */
+import debugPkg from "debug";
+import fetch from "node-fetch";
+
+import { APL, AuthData } from "./apl";
+
+const debug = debugPkg.debug("VercelAPL");
+
+export const TOKEN_VARIABLE_NAME = "SALEOR_AUTH_TOKEN";
+export const DOMAIN_VARIABLE_NAME = "SALEOR_DOMAIN";
+export const SALEOR_REGISTER_APP_URL = "SALEOR_REGISTER_APP_URL";
+export const SALEOR_DEPLOYMENT_TOKEN = "SALEOR_DEPLOYMENT_TOKEN";
```

Comment:
What do you think about some namespace object, like `VercelAplConfig` or something like this, so you will have:

```
import {VercelAplConfig} ...

VercelAplConfig.TOKEN_VARIABLE_NAME
```

This way we:
1. Are very explicit this is Vercel, not e.g. Netlify or something else (import paths are not good enough, because IDE is autogenerating them) - SDK will grow so this way we can mimic scoping/namespaces from other languages
2. Make it cohesive - put all related variables into single space where each of them can be found
3. Decrease number of exported symbols which is a problem scaling up